src/disco/components/Addon/styles.scss

```diff
@@ -156,6 +148,7 @@ $addon-padding: 20px;
     .copy {
       flex-grow: 1;
       padding: 20px 20px 15px;
+      word-break: break-all;
```

Comment:
do we really need this for all copy? I saw some words breaking where I thought it would look better if it could stay together...

![Alt text](https://monosnap.com/image/IsdaWs2EIXRn37skSiC99DmhB5nDlf.png)

(see word features)

Reply:
oops yeah, that's not nice..

Reply:
Can we use `word-wrap` instead of `break-word` . It should work I think 😬
https://codepen.io/raphaelgoetter/pen/JFqjz